README.md

```diff
@@ -390,16 +425,44 @@ Example assertions:
       }
 ```
 
-## Convention based file tests
```

Comment:
Need to explain how allOf() works somewhere in here:

public Requirement getRequirements(Configuration configuration) {
    return allOf(
            mutableTable(SINGLE_INT_COLUMN_PARTITIONEND_TEXTFILE, TABLE_NAME, LOADED),
            mutableTable(SINGLE_INT_COLUMN_PARTITIONED_ORC, TABLE_NAME, LOADED),
            mutableTable(SINGLE_INT_COLUMN_PARTITIONED_RCFILE, TABLE_NAME, LOADED),
            mutableTable(SINGLE_INT_COLUMN_PARTITIONED_PARQUET, TABLE_NAME, LOADED)
    );
}